zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/IncidentControllerTest.java

```diff
@@ -54,4 +61,61 @@ void shouldResolveIncident() {
 
     Mockito.verify(incidentServices).resolveIncident(1L);
   }
+
+  @Test
+  void shouldReturnNotFoundIfIncidentNotFound() {
+    // given
+    when(incidentServices.resolveIncident(anyLong()))
+        .thenReturn(CompletableFuture.completedFuture(new IncidentRecord()));
+
+    // when/then
+    webClient
+        .post()
+        .uri(INCIDENT_BASE_URL + "/1/resolution")
+        .accept(MediaType.APPLICATION_JSON)
+        .contentType(MediaType.APPLICATION_JSON)
+        .exchange()
+        .expectStatus()
+        .isNoContent();
+
+    Mockito.verify(incidentServices).resolveIncident(1L);
+  }
```

Comment:
🔧 Copy-paste mistake? This exists twice now, if I'm not completely confused 😄

Reply:
🙈